app/lib/event_bus.rb

```diff
@@ -28,8 +28,9 @@ def self.reset!
 # wiring
 
 # automation
-EventBus.subscribe_job :message_thread_created, Automation::MessageThreadCreatedJob
-EventBus.subscribe_job :message_created, Automation::MessageCreatedJob
+[:message_thread_created, :message_created, :message_draft_submitted].each do |event|
+  EventBus.subscribe_job event, Automation::EventTriggeredJob
```

Comment:
Ked uz robime toto, tak to nazvime asi podla toho co to aj robi `Automation::ApplyRulesForEventJob`

Reply:
premenoval som